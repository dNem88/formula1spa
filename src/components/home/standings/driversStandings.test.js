import React from 'react'
import renderer from 'react-test-renderer'
import DriverStandings from './DriversStandings'
import { MemoryRouter } from 'react-router-dom'


describe('Testing DriverStandings',() => {
    it('Render Component', () => {
        let snapshot = renderer.create(
            <MemoryRouter>
                <DriverStandings />
            </MemoryRouter>
        ).toJSON()
        expect(snapshot).toMatchSnapshot()
        expect(snapshot.type).toBe('div')
        expect(snapshot.props.className).toBe('main-container')
        expect(snapshot.children.length).toBe(3)
        let button = snapshot.children[2]
        expect(button.type).toBe('a')
        expect(button.props.href).toBe('/standings')
        expect(button.children[0]).toBe('VIEW FULL STANDINGS')
    })
})