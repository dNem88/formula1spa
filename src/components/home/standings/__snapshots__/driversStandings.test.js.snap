// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing DriverStandings Render Component 1`] = `
<div
  className="main-container"
>
  <div
    className="heading-container"
  >
    <h2>
      DRIVERS STANDINGS
    </h2>
  </div>
  <div
    className="standings-container"
  >
    <div
      className="lds-ring"
    >
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
  <a
    href="/standings"
    onClick={[Function]}
  >
    VIEW FULL STANDINGS
  </a>
</div>
`;
